persistence/src/commonMain/kotlin/com/wire/kalium/persistence/dao/conversation/ConversationDAO.kt

```diff
@@ -132,6 +132,11 @@ interface ConversationDAO {
     suspend fun getEstablishedSelfMLSGroupId(): String?
 
     suspend fun selectGroupStatusMembersNamesAndHandles(groupID: String): EpochChangesDataEntity?
+
+    /**
+     * TODO: Remove later when implementing general conversation cache in the app.
+     */
+    suspend fun observeConversationDetailsById(conversationId: QualifiedIDEntity): Flow<ConversationViewEntity?>
```

Comment:
Just to confirm:
The idea of not caching right now is avoiding having to change tests in `ConversationDAOTest`, right?

Cause other than that, it is exactly like `observeGetConversationByQualifiedID` (which has a worse name xD)

Reply:
It is to not introduce more variables to the new feature, we can Refactor later and unify once we are sure this is working as expected 🙂